src/cli.rs

```diff
@@ -17,18 +17,17 @@ use crate::runner;
 use crate::types::{CliArgs, GlobalConfig};
 use crate::version;
 
-static NAME: &str = "make";
-static VERSION: &str = env!("CARGO_PKG_VERSION");
-static AUTHOR: &str = env!("CARGO_PKG_AUTHORS");
```

Comment:
why this block change is needed? i hate going and manually changing things like version. won't it work with a secondary binary?